modules/govuk/manifests/apps/local_links_manager.pp

```diff
@@ -0,0 +1,19 @@
+# == Class: govuk::apps::local_links_manager
+#
+# App to manage the local links within GOV.UK so that
+# they can be used for local transactions.
+# https://github.com/alphagov/local-links-manager
+#
+class govuk::apps::local_links_manager(
+  $port = 3120,
```

Comment:
As of https://github.gds/gds/development/pull/299, 3120 is taken by Sidekiq Monitoring for Imminence.